pjmedia/src/pjmedia-codec/and_vid_mediacodec.cpp

```diff
@@ -1490,12 +1490,16 @@ static pj_status_t and_media_decode(pjmedia_vid_codec *codec,
         PJ_LOG(4,(THIS_FILE, "Decoder getOutputBuffer failed"));
         return status;
     }
-    len = write_yuv((pj_uint8_t *)output->buf,
-                    output->size,
-                    output_buf,
-                    and_media_data->dec_stride_len,
-                    and_media_data->prm->dec_fmt.det.vid.size.w,
-                    and_media_data->prm->dec_fmt.det.vid.size.h);
+    if (output->size >= and_media_data->prm->dec_fmt.det.vid.size.w *
```

Comment:
I think this check doesn't fix the overflow. Because `write_yuv` already checks `if (output->size < w * h * 3 / 2)`

I proposed a new fix:  Fix android video codec overflow #4617